forem/forem
# Implement Unified Embeds for Kotlin URLs

## What type of PR is this? (check all applicable)


## Description
This PR implements the [unified embed tag](https://github.com/forem/forem/issues/15099) for Kotlin embeds.

## Related Tickets & Documents
closes https://github.com/forem/forem/issues/15563

## QA Instructions, Screenshots, Recordings
Example Kotlin URLs:
https://pl.kotl.in/mCMciWl85
https://pl.kotl.in/owreUFFUG?theme=darcula
https://pl.kotl.in/Wplen1rPa?theme=darcula&from=6&to=7
https://pl.kotl.in/YzklsCOlF?theme=darcula&readOnly=true

- As a user, create a Kotlin liquid tag using this format: `{% embed <kotlin-url> %}`. The Liquid Tag should render properly.
- Also create Kotlin liquid tag using the old method: `{% kotlin <kotlin-url> %}`. This should also work, as I am leaving the old implementation in place for now.

### UI accessibility concerns?
None

## Added/updated tests?


## [Forem core team only] How will this change be communicated?
Considering the project as a whole, I shall update the Liquid Tag documentation at the end. Since I am not changing the current implementation, the present documentation is still valid. Plus, I will avoid updating the docs piecemeal, and risk confusing users.

## [optional] Are there any post deployment tasks we need to perform?
None
